src/main/java/Duke.java

```diff
@@ -1,10 +1,63 @@
+import java.io.IOException;
+
+import duke.command.Command;
```

Comment:
Perhaps you could have put an empty space from imports that are from different packages?